coding-concepts.js

```diff
@@ -48,9 +87,23 @@ const myCodingSkills = {
   versionControl: "GitHub"
 }
 // console.log(myCodingSkills.languages[0])
+// return =>JavaScript
+
+// console.log(myCodingSkills.languages[1])
+// return =>Ruby
+
+// console.log(myCodingSkills.databases)
+// return =>PostgreSQL
+
+// a) Your answer:The output should be JavaScript. It is the value of the 0th index within the languages section of myCodingSkills. The console.log is calling the myCodingClass and specifically the 0th (or first) index of languages. 
 
-// a) Your answer:
-// b) Verify and explain:
+// Expected return =>JavaScript
+
+// b) Verify and explain:I was correct! The console outputted JavaScript. I played around with the console.log statement to receive different outputs.
```

Comment:
Excellent work in this section!!  Happy to see that you are not only verifying the answers by playing around with the code to try to get different results as well!  This will continue to deepen your understanding!  👏